content/webapp/pages/page.js

```diff
@@ -56,13 +56,22 @@ export class Page extends Component<Props> {
       ) : null
     ) : null;
 
+    const hiddenBreadcrumbPages = ['X5amzBIAAB0Aq6Gm', 'X5aomxIAAB8Aq6n5'];
```

Comment:
Could you explain this to me? Perhaps we could stick this on the prismic model?

Reply:
I wanted to not show the `visit-us` breadcrumb, but I did want to keep the space that would otherwise be there when there are breadcrumbs, so I output a zero-width space character. I appreciate that this isn't scalable, but I think the breadcrumbs are ripe for a bit of a cleanup as well.

Reply:
there is a place called hardcoded-id in the prismic model, to make sure we can remove and clean up later etc..